oozie-to-airflow/mappers/mapreduce_mapper.py

```diff
@@ -92,7 +88,7 @@ def convert_to_text(self) -> str:
                 ),
             ),
         ]
-        relations = [Relation(from_task_id=self.name + "_prepare", to_task_id=self.name)]
+        relations = [Relation(from_task_id=self.name + "-prepare", to_task_id=self.name)]
```

Comment:
```suggestion
        relations = [Relation(from_task_id=self.name + "_prepare", to_task_id=self.name)]
```
I prefer to stay with old task name.  I prefer to stay at the old name. This is not required in this change.

Reply:
I do not care about the old name, but it has significantly reduced the number of conflicts that this change causes.